hack/dind-cluster.sh

```diff
@@ -95,6 +96,14 @@ function start() {
   echo "OPENSHIFT_ADDITIONAL_ARGS='${additional_args}'" > "${config_root}/additional-args"
   copy-runtime "${origin_root}" "${config_root}/"
 
+  # dockershim is default and doesn't need an enpoint path
+  runtime_endpoint=
+  if [[ "${container_runtime}" = "crio" ]]; then
+    runtime_endpoint="/var/run/crio.sock"
+  fi
+  echo "OPENSHIFT_CONTAINER_RUNTIME=${container_runtime}" > "${config_root}/container-runtime"
```

Comment:
There's not really any reason other than historical accident for putting every config option into a separate file... we could just squash them all into one